users/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+
+GENDER_SELECTION = [
+    ('M', 'Male'),
+    ('F', 'Female'),
+    ('NS', 'Not Specified'),
+]
+
+class User(AbstractUser):
+    gender = models.CharField(max_length=20, choices=GENDER_SELECTION)
+    first_name = models.CharField(max_length=30)
+    last_name = models.CharField(max_length=30)
```

Comment:
Here you can use `max_length=2` instead of 20, the saved value is the first one ('M', 'F' and 'NS'), 

I would recommend adding the values as inside the `User` class and as constants like in the [student example in the docs](https://docs.djangoproject.com/en/3.1/ref/models/fields/#choices), that way if you need to check for gender at some point you can do something like `if user.gender == User.MALE:` .

```suggestion
class User(AbstractUser):
    MALE = 'M'
    FEMALE = 'F'
    NOT_SPECIFIED = 'NS'
    GENDER_SELECTION = [
        (MALE, 'Male'),
        (FEMALE, 'Female'),
        (NOT_SPECIFIED, 'Not Specified'),
    ]

    gender = models.CharField(max_length=2, choices=GENDER_SELECTION)
    first_name = models.CharField(max_length=30)
    last_name = models.CharField(max_length=30)
```

Or better yet use the newer way using [enumeration types](https://docs.djangoproject.com/en/3.1/ref/models/fields/#enumeration-types)

```suggestion
class User(AbstractUser):

    class Gender(models.TextChoices):
        MALE = 'M', _('Male')
        FEMALE = 'F', _('Female')
        NOT_SPECIFIED = 'NS', _('Not specified')
        
    gender = models.CharField(max_length=2, choices=Gender.choices)
    first_name = models.CharField(max_length=30)
    last_name = models.CharField(max_length=30)
```